pkg/controller/csi-driver-lvm/actuator.go

```diff
@@ -337,6 +358,48 @@ func (a *actuator) controllerObjects() ([]client.Object, error) {
 								{MountPath: "/csi", Name: "socket-dir"},
 							},
 						},
+						{
+							Name:            "csi-driver-lvm-controller",
+							Image:           csiDriverLvmControllerImage.String(),
+							ImagePullPolicy: pullPolicy,
```

Comment:
It would be really cool if the pull policy would be configurable somehow. When running pull requests for dev this would make it a lot easier.